### DISCUSSION TITLE
Typeable field for adding data managers

### DISCUSSION THREAD

**Discussion by mossmatters** [Symbiota/community-board#248] [Wish List]
When I go to add a new data manager to my portal, I need to work through the entire list of everyone who has ever made an account on SEINet to update portals: 

<img width="599" alt="Screen Shot 2022-11-03 at 2 41 04 PM" src="https://user-images.githubusercontent.com/1557809/199818554-d2625e98-2285-4ff8-a248-aeedc324a4ed.png">

This is a small nuisance when working on a PC as I can type in the field and get close to the individual - though there may be an issue with identical names. However, on mobile devices, I need to scroll through the whole list. An improvement to this field would be to treat it as if it were a field that I could type in rather than a drop-down menu. 

A secondary improvement could be to have new accounts select one or more collections they would like to be associated with, at sign up. Then there could be a smaller drop-down list for accounts.

**Comment by themerekat** [comment]
@arbolitoloco would this be fixed in the new responsive design?
